src/status_im2/contexts/chat/messages/content/link_preview/view.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im2.contexts.chat.messages.content.link-preview.view
+  (:require
+    [quo2.core :as quo]
+    [utils.re-frame :as rf]))
+
+(defn- nearly-square?
+  [{:keys [width height]}]
+  (let [ratio (/ (max width height)
+                 (min width height))]
+    (< (Math/abs (- ratio 1)) 0.1)))
```

Comment:
Maybe would be worth to check for 0 value on width and height to prevent dividing by zero and cause the start of the apocalypse? Is that a possible case? (not the apocalypse, I was joking, but receiving 0 on width or height values :) )?

Reply:
Good catch @briansztamfater. In theory they should never be zero because the width and height are calculated from Go's core `Image` type, but we never know, so I'll update this function accordingly.

Reply:
Done :) I also added some unit tests. Thanks for the feedback.